src/containerapp/azext_containerapp/tests/latest/test_containerapp_commands.py

```diff
@@ -931,8 +931,8 @@ def test_containerapp_dev_service_binding_customized_keys_yaml_e2e(self, resourc
                 JMESPathCheck('properties.template.serviceBinds[1].customizedKeys', None),
             ])
 
-        self.cmd('containerapp service redis delete -g {} -n {} --yes'.format(env_rg, redis_ca_name), expect_failure=False)
-        self.cmd('containerapp service postgres delete -g {} -n {} --yes'.format(env_rg, postgres_ca_name), expect_failure=False)
+        self.cmd('containerapp add-on redis delete -g {} -n {} --yes'.format(env_rg, redis_ca_name), expect_failure=False)
```

Comment:
if you set target version to 2.59.0, then the test cases should keep until azurecli upgrades to 2.59.0

Reply:
Dup tests.